.changeset/seven-penguins-fly.md

```diff
@@ -0,0 +1,33 @@
+
+The Salt Splitter leverages the popular open-source library react-resizable-panels, whilst making adjustments to component names, adding custom styling, and introducing extra functionality to better align with the rest of the components present in our design system.
+
+## Remapping
+
+| salt-ds                                      | react-resizable-panels                       |
+| -------------------------------------------- | -------------------------------------------- |
+| `<Splitter />`                               | `<PanelGroup />`                             |
+| `<SplitPanel />`                             | `<Panel />`                                  |
+| `<SplitHandle />`                            | `<PanelResizeHandle />`                      |
+| `SplitterProps["orientation"]["vertical"]`   | `PanelGroupProps["direction"]["horizontal"]` |
+| `SplitterProps["orientation"]["horizontal"]` | `PanelGroupProps["direction"]["vertical"]`   |
+
+## Overridden Functionality
+
+`SplitterProps["orientation"]` - Replaced the `direction` prop with `orientation` to better align with the rest of the components present in our design system. The orientation prop takes one of two options `horizontal` and `vertical`. Keep in mind that they are the opposite of the base library's `direction` prop.
+
+## New Functionality
+
+`SplitterProps["appearance"]` - Change the appearance of the Splitter component. The appearance prop here acts similarly to `Button["appearance"]`. It can take one of two options `bordered` (default) and `transparent`. The bordered options will add a border on every nested `<SplitHandle />`, while the transparent option will make the handle transparent, thus letting through the background color of the parent component.
+
+`SplitPanelProps["variant"]` - Change the background color of the SplitPanel component. The variant prop takes one of three options - `primary` (default), `secondary`, `tertiary`. Follows the same behavior as our Panel component.
+
+`SplitHandle["variant"]` - Change the background color of the SplitHandle component. It behaves similarly to the `SplitPanelProps["variant"]` prop and it is meant to be used in combination with it. The variant prop can take one of three options - `primary` (default), `secondary`, `tertiary`. Follows the same behavior as our Panel component.
+
+`SplitHandle["border"]` - Override the border placement of the SplitHandle component. The border prop takes one of seven options - `top`, `bottom`, `left`, `right`, `top-bottom`, `left-right`, `none`. The default value is computed based on the orientation and appearance of the parent Splitter component. If the parent splitter's orientation is `horizontal` and the appearance is `bordered`, the default value will be `top-bottom`. If the orientation is `vertical` and the appearance is `bordered`, the default value will be `left-right`. If the appearance is `transparent`, the default value will be `none`.
```

Comment:
I don't think this needs to be in the changelog, realistically to consumers all they want to know is that Splitter is added

Reply:
Moved this files contents in a dedicated docs page in storybook